input/config.json

```diff
@@ -138,53 +138,61 @@
       "itemMappings": {
         // Black Lion Exclusives Chest
         "82450": [
-          7225,
-          7233,
-          7241,
-          7782,
-          7800,
-          7804,
-          7941,
-          7975,
-          7985,
-          8115,
-          8121,
-          8122
+          // Sworn Zaishen Helm
+          7225,7233,7241,
+          // Fallen Balthazar Helm
+          7782,7800,7804,
+          // Svanir Gauntlets
+          7941,7975,7985,
+          // Exalted
+          8115,8121,8122,8617,8618,8621,
+          // Seven Reapers
+          9376,9384,9447,9890,9891,9894,10098,10100,10106,10124,10129,10131,
+          // Ebon Vanguard Elite Shoulders
+          9577,9580,9581
         ],
+        // Southsun Remembrance Package
+        "84207": [996,997,998,1137,1143,1141],
+        // Chest of Gloves of the Obedient
+        "80086": [7077,7115,7121],
```

Comment:
What is the reasoning in adding this/other container configuration? The purpose of these mappings to enable prices to be put on items based on containers they are when those containers are available on the TP - but this isn't the case for the raid armour containers, and containers *should* be working correctly for merchants/vendors already?

Reply:
I saw some raid items weren't retrieving their prices, but they are now, so might be a cache thing due to recent changes?

Seems they're all fine now, except for the Dhuum heavy boots, those seem to be failing for some reason.

Reply:
Yeah, I broke all the prices when I added emotes - not surprising something shook lose because it required some fundamental changes to support them (non-numeric ids).

I'll have a look into the heavy boots, it is probably a wiki crawling thing, maybe bad cached data.

Reply:
Consider the raid containers removed then, pointless to have them there if they're just for trading post prices :D.

Reply:
Dhuum boots seem fixed now after re-downloading the relevant pages.